### DISCUSSION TITLE
数学公式字体和西文字体长得不一样怎么办？

### DISCUSSION THREAD

**Discussion by AgnesLee97** [tuna/thuthesis#727] [Q&A] [answered]
《研究生学位论文写作指南》第14页要求表达式采用 Cambria Math 或 Times New Roman 体，而模板编译的时候不管选stix | xits | libertinus 中的哪个，都和times new roman在26个字母的表示上有明显区别。
比如stix和xits的小写字母w，是图1和图2那样的；times new roman是图3那样的。
而我需要在论文的图里面标注这个w字母，在photoshop中用times new roman只能是图3这种效果，公式里却打不出来图中times new roman的效果，应该怎么办呢？
![image](https://user-images.githubusercontent.com/71933090/153706091-0fc2a03c-6efb-4bfe-810a-2079a64ce7d2.png)
![image](https://user-images.githubusercontent.com/71933090/153706095-f9bab492-a4e8-4eca-ba0b-1dc73c0d63d7.png)
![image](https://user-images.githubusercontent.com/71933090/153706086-c8d78f83-e231-4ead-8de1-dd1015dbc1de.png)

**Comment by zepinglee** [comment, marked as answer]
我跟研究生院的老师讨论过这个问题，STIX 和 XITS 都是 Times-compatible 的数学字体，可以放心使用。另外 Times New Roman 没有“官方”的数学字体，如果直接用 `mathspec` 宏包将其用与数学字体会导致一些 metric 的问题。

**Reply by AgnesLee97** [reply]
非常感谢！

**Reply by zepinglee** [reply]
有个 `mathspec` 宏包允许在数学符号使用普通的字体，但是对于稍微复杂一点的数学公式效果就会很差。

```latex
\documentclass{article}

\usepackage{mathspec}
\setallmainfonts{Times New Roman}

% \usepackage{fontspec}
% \usepackage{unicode-math}
% \setmainfont{Times New Roman}
% \setmathfont{STIXTwoMath-Regular.otf}

\begin{document}
\begin{equation}
  f(x^i, x^j)
\end{equation}
\end{document}
```
<img width="186" alt="Screen Shot 2022-02-18 at 22 35 26" src="https://user-images.githubusercontent.com/12290822/154702643-2d27e6f1-13b0-4bfe-ab05-3985369a0fea.png">

`unicode-math` 与 SITX Two Math 的效果：
<img width="195" alt="Screen Shot 2022-02-18 at 22 36 09" src="https://user-images.githubusercontent.com/12290822/154702792-64f4903b-cc76-45d9-8b27-53d1e921acbe.png">

**Reply by AgnesLee97** [reply]
哦哦，这个效果确实不太好，还是用sitx two math好了，非常感谢！
